Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:17-alpine
+
+WORKDIR /app
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 3000
+
+CMD [ "npm","start" ]
```

Comment:
There is no point in running the web app using `npm start` since we can use `npm start` and test the app directly. 

So I think we can remove this file and only use the production file.

Reply:
Pushed the changes